src/Blazored.Toast/CountdownTimer.cs

```diff
@@ -1,57 +1,72 @@
-﻿namespace Blazored.Toast;
+﻿using System.Threading;
```

Comment:
Nit pick here, but can you reinstate the line between each method. It makes things much harder to read when methods are not separated.

Reply:
i am not sure what you ment by this, but using System.Threading is removed from here, no need from it.

Reply:
I was referring to the fact there were no line breaks between methods in the class. Sorry, the placement of my comment confused things.